Say Their Names/Source/Frameworks/ServiceFrameworks.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ServiceFrameworks.swift
+//  Say Their Names
+//
+//  Created by evilpenguin on 5/31/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+import Firebase
+
+// MARK: - ServiceFrameworks
+
+typealias ServiceFrameworks = Service
+extension ServiceFrameworks {
```

Comment:
Whats the benefit of using a `typealias` here?

Reply:
There is none outside of abstraction, and not just have `extension class` everywhere.

Reply:
If you think this makes things easier from a readability pov then let's go ahead and keep it. I'll leave it up to you

Reply:
We can always remove it later. Simple refactor in the feature.